inst/rmarkdown/templates/pihph/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,97 @@
+
+# Heading level 2
+
+Quotations of under 25 words should be included in the running text \enquote{as a wise person once said} (with an associated full reference, including page numbers). All references should follow the normal Author (date, page number) system.  Longer quotations should be set out as follows, which represents the words of Bloggs (1937, 23).
+
+
+> This is the format for a quotation of 25 words or more, with indentation of 0.7 cm throughout the quotation on both sides and a font size of 10 points (this is set up in the template) and a reference in the text above it, unless there is a good reason to give the reference elsewhere.
+
+All examples and anything that is not a table or figure should be given a number for reference, as normal. You can set out your examples, diagrams and other similar items in any way that you think sensible (within the general constraints of this template). We recommend the expex package for example, but you are welcome to use any others. Put the example number in brackets, but if you use subexamples the numbering is up to you. The numbers for examples should not be indented.
+
+```{=latex}
+\ex\begin{tabular}[t]{lll}
+     /ɛɡzampl/ & [ɛɡzámpl̩] & `example'
+   \end{tabular}
+\xe
+
+\ex\labels\begin{tabular}[t]{>{\tl}llll} % or use \pex
+& /tu e/ & [tʰʉː eː] & `two a' \\
+& /tu bi/ & [tʰʉː biː] & `two b'
+\end{tabular}
+\xe
+```
+
+If you need to include translations for examples, use single quotation marks. Do not use bold or underlining anywhere in an article (apart from where required by the template in headings and the like). Use italics for linguistic examples in the running text, for the titles of publications and for any kind of emphasis. If you encounter any problems with setting out your examples and similar things, contact us for advice (pihph@\hspace{0pt}mlist.is.ed.ac.uk).
+
+Captions for figures and tables should be set out below the figure or table. We recommend, but do not insist, that you avoid vertical rules in your tables (we recommend the booktabs package). Otherwise, the setting of figures and tables is also up to you. Everything should be placed in your article where you would like it to appear in the published PDF. Do not place anything at the end of an article unless it clearly belongs in an appendix (that is, it consists of material that a casual reader will not want to consult, but may be of interest to specialists). You will be able to host data sets, statistical data, scripts, and the like separately (within reason) if you would like to make them available in connection with an article. Contact us if you would like to discuss this (pihph@\hspace{0pt}mlist.is.ed.ac.uk).
+
+\begin{figure}[h]
+  \centering
+  \rule{.1pt}{2cm} \rule{2cm}{.1pt} \rule{.1pt}{2cm}
+  \caption{Caption}
+  \label{fig:example}
+\end{figure}
```

Comment:
Should we use 
````suggestion
```{=latex}
\begin{figure}[h]
  \centering
  \rule{.1pt}{2cm} \rule{2cm}{.1pt} \rule{.1pt}{2cm}
  \caption{Caption}
  \label{fig:example}
\end{figure}
```
````

as you did above ? 

you can also give an example of figure insertion using R code.

Reply:
Included `knitr::include_graphics()` instead